### DISCUSSION TITLE
Need help achieving my dream workflow with Todoist tasks

### DISCUSSION THREAD

**Discussion by jamie9090** [chhoumann/quickadd#469] [Q&A]
Sorry, not quite sure where to put this, but maybe someone could help? 

I basically use Todoist for tasks, but also idea capture with the latter is what I'm trying to get into Obsidian (and out of Todoist) automatically. 

I would love to be able to do the following every time Obsidian launches:

- Run everytime Obsidian launches (so it's relatively automated)
- Pull all Tasks from a specific Project (likely I would have multiple projects; so maybe this would be a script per project?)
- Pull the Task Descriptions as well
- Add to the bottom of a specific file
- Tick off all Tasks once added to Obsidian

I feel like QuickAdd can do 90% of this (and probably 100% if I was more code-literate)

Thank you!!

**Comment by chhoumann** [comment]
Hi @jamie9090 

Did you see this?
https://quickadd.obsidian.guide/docs/Examples/Capture_FetchTasksFromTodoist

It does much of what you're looking for. The script would need some modification to fit your use-case, though.
